### PR TITLE
Combine ts_daily_summary into write_traffic_stats

### PR DESCRIPTION
**Pull request by PSUdaemon**
This merges ts_daily_summary into write_traffic_stats so now there is one process to run instead of two.

Also, this updates write_traffic_stats to handle the new cdnName changes.

### CODE REVIEW

**Comment by PSUdaemon** [comment]
After offline discussion with @dneuman64 I made the entire package more generalized as "traffic_stats" and fixed up a few things like sending stats before we shutdown and making config reloadable.

File: traffic_stats/traffic_stats.go [hunk 1]
```diff
@@ -122,32 +154,211 @@ func main() {
...
+					continue
+				}
+				max = FloatMax(max, kbps)
+				duration := sampleTime.Unix() - prevtime.Unix()
+				bytesServed += float64(duration) * kbps / 8
+				prevtime = sampleTime
+			}
+			maxGbps := max / 1000000
+			bytesServedTb := bytesServed / 1000000000
+			log.Debugf("max gbps for cdn %v = %v", cdn, maxGbps)
```
Comment (dneuman64) [inline traffic_stats/traffic_stats.go]: Can we make this log.Infof?

File: traffic_stats/traffic_stats.go [hunk 2]
```diff
@@ -122,32 +154,211 @@ func main() {
...
+			log.Debugf("Tbytes served for cdn %v = %v", cdn, bytesServedTb)
```
Comment (dneuman64) [inline traffic_stats/traffic_stats.go]: this one log.Infof too

File: traffic_stats/traffic_stats.go [hunk 3]
```diff
@@ -248,7 +464,7 @@ func calcMetrics(cdnName string, url string, cacheGroupMap map[string]string, co
 	} else if strings.Contains(url, "DsStats") {
 		err = calcDsValues(rascalData, cdnName, sampleTime, config)
 	} else {
-		log.Info("Don't know what to do with ", url)
+		log.Debug("Don't know what to do with ", url)
```
Comment (dneuman64) [inline traffic_stats/traffic_stats.go]: lets make this a warn

**Pull request merged**